index.js

```diff
@@ -98,17 +104,24 @@ const platformConfigs = {}
 
 // Handle platform specific overrides and write configs to disk
 for (const platform of platforms) {
-    const platformConfig = JSON.parse(JSON.stringify(defaultConfig))
+    let platformConfig = JSON.parse(JSON.stringify(defaultConfig))
     const overridePath = `${OVERRIDE_DIR}/${platform}-override.json`
 
+    // Use extension config as the base for browser configs
+    // Extension comes first in the list of platforms so its config should be defined
+    // in the platformConfigs array
+    if (platform.includes(BROWSERS_SUBDIR)) {
+        platformConfig = JSON.parse(JSON.stringify(platformConfigs.extension))
```

Comment:
This relies on the sort order of ./platforms being exactly as it is.

Can you file a follow up to fix this and/or add a comment to the platforms.js to indicate ordering matters.